nuitka/BytecodeCaching.py

```diff
@@ -24,6 +24,7 @@
 import os
 
 from nuitka.importing.Importing import getPackageSearchPath, isPackageDir
+from nuitka.nodes.ImportNodes import ModuleUsageAttempt
```

Comment:
The `ModuleUsageAttempt` should be moved to `nuitka.importing.Importing`, it normally is never necessary to import nodes to do something with data received.